js/windowMgr.js

```diff
@@ -365,6 +355,14 @@ function doCreateMainWindow(initialUrl, initialBounds) {
                         });
                     });
 
+                    // In case we navigate to an external link from inside a pop-out,
+                    // we open that link in an external browser rather than creating
+                    // a new window
+                    browserWin.webContents.on('new-window', (childEvent, childWinUrl) => {
```

Comment:
make sure event is cleaned up when pop-out is closed.
also same above for 'crashed' event
or at least ensure no leakage occurs.

Reply:
done.